protos/PublicDefs.proto

```diff
@@ -244,8 +243,9 @@ message UserInfo {
   uint32 uid = 1;
   string name = 2;
   string account = 3;
-  repeated AllowedPartitionQos allowed_partition_qos_list = 4;
-  AdminLevel admin_level = 5;
+  bool enable = 4;
```

Comment:
Does it break compatible？

Reply:
The interface on the back end will be modified synchronously. The grpc interface has to be modified because the user's data structure changes